internal/operator/clusterutilities.go

```diff
@@ -157,8 +158,8 @@ type DeploymentTemplateFields struct {
 	ContainerResources       string
 	NodeSelector             string
 	ConfVolume               string
-	CollectAddon             string
-	CollectVolume            string
+	ExporterAddon            string
+	ExporterVolume           string
```

Comment:
I don't think this is used?

Reply:
No, I missed that reference to 'ExporterVolume' during cleanup. It's removed now, thanks!